src/componenet/Page.jsx

```diff
@@ -0,0 +1,49 @@
+import { useState } from 'react';
+
+import Todozone from './Todozone';
+
+export default function Page() {
+  const [state, setState] = useState({
+    todos: [],
+    text: '',
+  });
+
+  const { todos, text } = state;
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    const copyState = [...todos];
+    copyState.push(text);
+    setState({
+      todos: copyState,
+      text: '',
+    });
```

Comment:
```suggestion
    setState({
      todos: [...todos, text],
      text: '',
    });
```

처럼 해주면 더 간단할 것 같아요 ;)